When changing the README.md only without the init/eessi_defaults file to default, the CI Tests for consistency of README.md fails with :
```
Run source init/eessi_defaults
  source init/eessi_defaults
  grep "${EESSI_CVMFS_REPO}" README.md
  shell: /usr/bin/bash -e {0}
Error: Process completed with exit code 1.
```

--- a/init/eessi_defaults
+++ b/init/eessi_defaults
@@ -8,8 +8,7 @@
 # license: GPLv2
 #
 
-export EESSI_CVMFS_REPO="${EESSI_CVMFS_REPO_OVERRIDE:=/cvmfs/software.eessi.io}"
-# export EESSI_CVMFS_REPO="${EESSI_CVMFS_REPO_OVERRIDE:=/cvmfs/pilot.nessi.no}"
+export EESSI_CVMFS_REPO="${EESSI_CVMFS_REPO_OVERRIDE:=/cvmfs/pilot.nessi.no}"
 export EESSI_VERSION="${EESSI_VERSION_OVERRIDE:=2023.06}"
 # use archdetect by default, unless otherwise specified
 export EESSI_USE_ARCHDETECT="${EESSI_USE_ARCHDETECT:=1}"

